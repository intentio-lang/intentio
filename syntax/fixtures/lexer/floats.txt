1.0
1.0e92
10e_6
1_________.0_______
1_.0
10_e5
35e+5
7e-5
8E5
