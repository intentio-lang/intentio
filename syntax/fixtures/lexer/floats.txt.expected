[
  {
    "_text": "1.0",
    "_ty": "TFloat"
  },
  {
    "_text": "1.0e92",
    "_ty": "TFloat"
  },
  {
    "_text": "10e_6",
    "_ty": "TFloat"
  },
  {
    "_text": "1_________.0_______",
    "_ty": "TFloat"
  },
  {
    "_text": "1_.0",
    "_ty": "TFloat"
  },
  {
    "_text": "10_e5",
    "_ty": "TFloat"
  },
  {
    "_text": "35e+5",
    "_ty": "TFloat"
  },
  {
    "_text": "7e-5",
    "_ty": "TFloat"
  },
  {
    "_text": "8E5",
    "_ty": "TFloat"
  }
]
